SCons/Node/FSTests.py

```diff
@@ -3542,6 +3542,58 @@ def __call__(self, target, source, env, executor=None):
         dir.prepare()
 
 
+if sys.platform != 'win32' and hasattr(os, 'symlink'):
+    class CleanSymlinksTestCase(_tempdirTestCase):
+
+        def test_cleans_symlinks(self):
+            """Test the prepare() method will cleanup symlinks."""
+            with open("foo", "w") as foo:
```

Comment:
Please change to use something like this (see line 1556)
`        fp = open(test.workpath('can_not_remove'))`
This creates the files in the temporary directory created to run the test.